source/index.html

```diff
@@ -1,12 +1,211 @@
 <!DOCTYPE html>
-<html lang="ru">
+<html class="page" lang="ru">
   <head>
     <meta charset="utf-8">
     <title>HTML Academy: Кэт энерджи</title>
   </head>
-  <body>
+  <body class="page__body">
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
+    <header class="main-header">
+
+      <a class="logo" href="#">
+        <img class="logo__logo-image" src="../source/img/logo.svg" width="202" height="auto" alt="Логотип Кэт Энерджи.">
+      </a>
+
+      <nav class="main-nav">
+        <button class="main-nav__toogle" type="button">
+          <span class="visually-hidden">Открыть меню</span>
+        </button>
+        <div class="main-nav__wrapper">
+          <ul class="main-nav__list">
+            <li class="main-nav__item main-nav__item--current">
+              <a class="main-nav__link" href="#">
+                Главная
+              </a>
+            </li>
+            <li class="main-nav__item">
+              <a class="main-nav__link" href="catalog.html">
+                Каталог продукции
+              </a>
+            </li>
+            <li class="main-nav__item">
+              <a class="main-nav__link" href="form.html">
+                Подбор программы
+              </a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+
+    </header>
+
+    <main class="main-container">
+
+      <header class="main-container__header">
+        <h1 class="main-container__title">Функциональное питание для котов</h1>
+        <p class="maiin-container__slogan slogan">
+          Занялся собой? Займись котом!
+        </p>
+        <a class="main-container__button button" href="form.html">Подобрать программу</a>
+      </header>
+
+      <section class="programs">
+        <div class="programs__wrapper">
+          <h2 class="visually-hidden">Программы питания.</h2>
+          <ul class="programs__list">
+            <li class="programs__item">
+              <h3 class="programs__title">Похудение</h3>
+              <p class="programs__description">
+                Ваш кот весит больше собаки и почти утратил способность
+                лазить по деревьям? Пора на диету! Cat Energy Slim поможет
+                вашему питомцу сбросить лишний вес.
+              </p>
+              <a class="programs__link--slim program-slim" href="#">Каталог slim</a>
```

Comment:
Модификатор без основного класса, возможно должно быть так?
```suggestion
              <a class="programs__link programs__link--slim program-slim" href="#">Каталог slim</a>
```
Исправь подобные ошибки дальше